pkg/virt-api/rest/subresource_test.go

```diff
@@ -933,15 +935,15 @@ var _ = Describe("VirtualMachineInstance Subresources", func() {
 						VolumeSource: &v1.HotplugVolumeSource{},
 					},
 				},
-				"[{ \"op\": \"test\", \"path\": \"/spec/volumes\", \"value\": [{\"name\":\"existingvol\",\"persistentVolumeClaim\":{\"claimName\":\"testpvcdiskclaim\"}}]}, { \"op\": \"test\", \"path\": \"/spec/domain/devices/disks\", \"value\": [{\"name\":\"existingvol\"}]}, { \"op\": \"replace\", \"path\": \"/spec/volumes\", \"value\": [{\"name\":\"existingvol\",\"persistentVolumeClaim\":{\"claimName\":\"testpvcdiskclaim\"}},{\"name\":\"vol1\"}]}, { \"op\": \"replace\", \"path\": \"/spec/domain/devices/disks\", \"value\": [{\"name\":\"existingvol\"},{\"name\":\"vol1\"}]}]",
+				"[{\"op\":\"test\",\"path\":\"/spec/volumes\",\"value\":[{\"name\":\"existingvol\",\"persistentVolumeClaim\":{\"claimName\":\"testpvcdiskclaim\"}}]},{\"op\":\"test\",\"path\":\"/spec/domain/devices/disks\",\"value\":[{\"name\":\"existingvol\"}]},{\"op\":\"replace\",\"path\":\"/spec/volumes\",\"value\":[{\"name\":\"existingvol\",\"persistentVolumeClaim\":{\"claimName\":\"testpvcdiskclaim\"}},{\"name\":\"vol1\"}]},{\"op\":\"replace\",\"path\":\"/spec/domain/devices/disks\",\"value\":[{\"name\":\"existingvol\"},{\"name\":\"vol1\"}]}]",
```

Comment:
I would personally use the patchset type also in this case, because we can avoid this formatting and ordering of the type

Reply:
I tried to give it a try, but the result is that each `Entry` became incredibly huge and the readability is seriously affected. I would leave it as is for the moment. Maybe we can try to address it in a separate PR. WDYT?

Reply:
I'm afraid that we'll forget about it. Any reasons to not handle it in this PR? i think you can define some functions that hides the values assignment. For example:
```golang

type vol struct {
    name string
    pvc string
}

func testAndReplaceVols(oldVols, newVols []vol) []byte{
     oldVol []v1.Volume
     oldDisk []v1.Disk
     newVol []v1.Volume
     newDisk []v1.Disk
     .... create the 2 volumes set and the patch
}
```
then in the entry you have:
```golang
testAndReplaceVols([]vol{{"existing","testpvcdiskclaim }}, ([]vol{{"existing","testpvcdiskclaim }, {"vol1"}}
```

Reply:
Sure, will do soon. Thanks

Reply:
From a second look, it appears that my worries were unfounded.
I opted for the patchSet without using the `testAndReplaceVols()`, because it is only two usage for the volumes.

Reply:
I personally like more this version with the struct even if the Entry is larger, thanks!

Reply:
Yeah I agree, I was scared previously if there were a high number of those larger `Entry`.
Thank you!